client/src/components/payment/payment-confirm.vue

```diff
@@ -69,24 +67,8 @@ export default class PaymentConfirm extends Vue {
     required: false
   }
   @Prop(String) name: string
-  // @Prop(Boolean) priorityRequest: boolean
-  // @Prop(Object) payment: any
-  @Prop({ default: {
```

Comment:
This was a left-over prop decoration, which was being attached to fetchError below incorrectly and causing an error.